src/app/core/components/game-statistics/game-statistics.component.html

```diff
@@ -5,9 +5,17 @@
   </div>
   <span class="stat-container main">
     <span class="total-amount">
-      {{ statistics.amountInCollection }}
+      {{ statistics.amountInCollection | number: '1.0-0' : 'en-US' }}
```

Comment:
en-US should be the default local right now. Why did you add it ?

Reply:
Wasn't sure if angular defaults to en-US or bases it on browser language so I wanted to be on the safe side